index.html

```diff
@@ -16,9 +16,29 @@
 
 <body>
 
-  
+    <div class ="nav">
```

Comment:
```suggestion
       <header class ="nav">
       <div>
        <h1>Fun Bus</h1>
        <nav>
          <a href="#">Home</a>
          <a href="#">About Us</a>
          <a href="#">Blog</a>
          <a href="#">Contact</a>
        </nav>
        </div>
      </header>
```
With this set up you can set the .nav class to width 100% of the screen, and set the div to a width: 100% and max-width: 800px and margin: 0 auto and then use flex box to position it better